test/test_http11.rb

```diff
@@ -183,4 +183,30 @@ def test_horrible_queries
     end
 
   end
+
+  def test_13632_workaround
```

Comment:
If it's not defined and the bug is happened, than the test is supposed to fail. It's its purpose.